packages/api-clients/open-api/bffApi.yml

```diff
@@ -17736,6 +17739,27 @@ components:
           format: uuid
         tenantKind:
           $ref: "#/components/schemas/TenantKind"
+    EServiceTemplateRef:
+      type: object
+      additionalProperties: false
+      required:
+        - templateId
+        - templateVersionId
+        - templateName
+        - instanceId
+      properties:
+        templateId:
+          type: string
+          format: uuid
+        templateVersionId:
+          type: string
+          format: uuid
+        templateName:
+          type: string
+        instanceId:
+          type: string
+          format: uuid
```

Comment:
It is not a uuid
```suggestion
```

Reply:
Done with [1573a54](https://github.com/pagopa/interop-be-monorepo/pull/1598/commits/1573a5443410dc4497e84ad72ad46bfaee5b28b8)